### PR TITLE
Replace 'Icinga2' with 'Icinga 2'

### PR DESCRIPTION
**Pull request by gunnarbeutner**
The official spelling of the project name is 'Icinga 2'. This PR updates the Icinga 2 installation guide to replace 'Icinga2' with 'Icinga 2'.

### CODE REVIEW

File: docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ external_resources:
  - '[Official Icinga Documentation](https://www.icinga.com/docs/icinga2/latest/doc/01-about/)'
 ---
 
-![Install Icinga2 Monitoring on Debian 9](/docs/assets/icinga/Icinga2.jpg "Install Icinga2 Monitoring on Debian 9")
+![Install Icinga 2 Monitoring on Debian 9](/docs/assets/icinga/Icinga 2.jpg "Install Icinga 2 Monitoring on Debian 9")
```
Comment (GuessWhoSamFoo) [inline docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md]: This will break the image path

Comment (gunnarbeutner) [reply docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md]: Ouch, and I thought I had carefully reviewed what sed gave me. I guess I was wrong. :)

Comment (GuessWhoSamFoo) [reply docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md]: lgtm, will merge on green

Comment (gunnarbeutner) [reply docs/uptime/monitoring/install-icinga2-monitoring-on-debian-9.md]: Cool, thanks. 😊

**Pull request merged**